### PR TITLE
[TFLite 16x8] Leaky Relu reference kernel for activations in int16

### PR DESCRIPTION
**Pull request by wwwind**
This PR is one of steps to extend 8-bit quantization to support symmetric 16-bit activations - each activation is of type int16 and symmetric around zero. The weight tensor precision remains at 8-bit signed values. The bias is set to int64 precision.

In this PR we enable LEAKY_RELU kernel reference function.
Versioning is added.

### CODE REVIEW

**Comment by teijeong** [comment]
Hi Renjie, can you take a look?

File: tensorflow/lite/kernels/activations_test.cc [hunk 1]
```diff
@@ -108,10 +108,20 @@ class BaseActivationsOpModel : public SingleOpModel {
...
+        input.type == TensorType_INT16) {
       auto output_min = (input.min >= 0) ? input.min : input.min * alpha;
       auto output_max = (input.max >= 0) ? input.max : input.max * alpha;
-      output_ = AddOutput({input.type, {}, output_min, output_max});
+      if (input.type == TensorType_INT16) {
+        output_ = AddOutput({TensorType_INT16,
+                           {},
+                           output_min,
+                           output_max,
+                           1.0f / (std::numeric_limits<int16_t>::max() + 1),
```
Comment (renjie-liu) [inline tensorflow/lite/kernels/activations_test.cc:120]: is here trying to set the scale?

Comment (wwwind) [reply tensorflow/lite/kernels/activations_test.cc]: Hi @renjie-liu ! Thanks for pointing this out.  I corrected.

**Review by renjie-liu — DISMISSED** [review]

**Comment by gbaned** [comment]
@wwwind Can you please resolve conflicts? Thanks!

**Comment by wwwind** [comment]
Hi @renjie-liu , Could you please re-approve this PR ? I had to resolve a conflict.

**Review by renjie-liu — APPROVED** [review]

**Pull request merged**